pkgs/tools/inputmethods/ibus/default.nix

```diff
@@ -205,13 +229,15 @@ stdenv.mkDerivation rec {
     tests = {
       installed-tests = nixosTests.installed-tests.ibus;
     };
+    updateScript = nix-update-script { };
```

Comment:
The tests above shouldn't be run on `libOnly` builds, since they don't produce the install tests